sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -86,12 +85,11 @@ def run_pipeline(self, pipeline):
     proto_context = pipeline_context.PipelineContext(
         default_environment_url=self._docker_image)
     proto_pipeline = pipeline.to_runner_api(context=proto_context)
-    if self._runner_api_address:
-      for pcoll in proto_pipeline.components.pcollections.values():
-        if pcoll.coder_id not in proto_context.coders:
-          coder = coders.registry.get_coder(pickler.loads(pcoll.coder_id))
-          pcoll.coder_id = proto_context.coders.get_id(coder)
-      proto_context.coders.populate_map(proto_pipeline.components.coders)
+    for pcoll in proto_pipeline.components.pcollections.values():
```

Comment:
This runner_api_address thing was used as a boolean flag that was always true so I'm not sure what it's for.